clang/include/clang/Sema/Overload.h

```diff
@@ -1117,16 +1205,27 @@ class Sema;
     SmallVector<OverloadCandidate, 16> Candidates;
     llvm::SmallPtrSet<uintptr_t, 16> Functions;
 
-    // Allocator for ConversionSequenceLists. We store the first few of these
+    DeferredTemplateOverloadCandidate *FirstDeferredCandidate;
+    unsigned DeferredCandidatesCount : 8 * sizeof(unsigned) - 2;
```

Comment:
isn't this size possibly different on platforms, and thus makes the `32` on 1227 unreliable?

Reply:
The 32 on line 1227 is super arbitrary and not tied to the size of `unsigned`.
It did felt worth making bigger as we are now storing more than conversion sequences in the overload set.